### PR TITLE
Realign baseline/add additional `case-*` icons

### PR DESCRIPTION
**Pull request by danielbayley**
* camelCase
* snake_case
* kebab-case

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
I think it would be better for consistency to have kebab and snake case to use the same letter shapes, maybe something like this (3px radius for both instead of 4px for one and 2px for the other):
image

Not sure about the use cases for these though. Where does VSCode use icons like this?
(I'm guessing that's where these originate from :D )

File: icons/case-snake.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <line x1="7" x2="7" y1="10" y2="15" />
+  <circle cx="4.5" cy="12.5" r="2.5" />
+  <line x1="11" x2="13" y1="15" y2="15" />
+  <circle cx="19.5" cy="12.5" r="2.5" />
+  <line x1="17" x2="17" y1="8" y2="15" />
```
Comment (karsa-mistmere) [inline icons/case-snake.svg:16]:
```suggestion
  <path d="M7 10v5" />
  <circle cx="4.5" cy="12.5" r="2.5" />
  <path d="M11 15h2" />
  <circle cx="19.5" cy="12.5" r="2.5" />
  <path d="M17 8v7" />
```

File: icons/case-kebab.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="4.5" cy="12.5" r="2.5" />
+  <line x1="7" x2="7" y1="10" y2="15" />
+  <line x1="11" x2="13" y1="12" y2="12" />
+  <circle cx="19.5" cy="12.5" r="2.5" />
+  <line x1="17" x2="17" y1="8" y2="15" />
```
Comment (karsa-mistmere) [inline icons/case-kebab.svg:16]:
```suggestion
  <circle cx="4.5" cy="12.5" r="2.5" />
  <path d="M7 10v5" />
  <path d="M11 12h2" />
  <circle cx="19.5" cy="12.5" r="2.5" />
  <path d="M17 8v7" />
```

File: icons/case-camel.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="7" cy="12" r="3" />
+  <line x1="10" x2="10" y1="9" y2="15" />
+  <path d="M14 7h4c1.1 0 2 .9 2 2s-.9 2-2 2h-4V7z" />
+  <path d="M14 11h4.5c1.1 0 2 .9 2 2s-.9 2-2 2H14v-4z" />
```
Comment (karsa-mistmere) [inline icons/case-camel.svg:15]:
```suggestion
  <circle cx="7" cy="12" r="3" />
  <path d="M10 9v6" />
  <path d="M14 11h4.5a2 2 0 0 1 0 4H14V7h4a2 2 0 0 1 0 4" />
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Review by karsa-mistmere — APPROVED** [review]

**Comment by jguddas** [comment]
What about PASCAL_CASE?

**Comment by karsa-mistmere** [comment]
Do you mean PascalCase? 🤔

**Comment by danielbayley** [comment]
> What about *PascalCase?

@jguddas Would/could that be covered by https://lucide.dev/icons?search=case-upper? Difficult to express with only 2 letters…

**Comment by karsa-mistmere** [comment]
> > What about *SCREAMING_SNAKE_CASE?
> 
> @jguddas Would/could that be covered by https://lucide.dev/icons?search=case-upper? Difficult to express with only 2 letters…

I think @jguddas meant an icon depicting `A_B` (or some other letters), which is `UPPER_SNAKE_CASE`, also known as `SCREAMING_SNAKE_CASE`

**Comment by danielbayley** [comment]
> I think @jguddas meant an icon depicting … `UPPER_SNAKE_CASE`

I see. Well that one is a bit tricky because the upper case B can’t be made any smaller… But see what you think of my latest `push`ed changes… @jguddas @karsa-mistmere

**Review by ericfennis — APPROVED** [review]

**Comment by jguddas** [comment]
How about making them all align to the same baseline rather than just entering them vertically?

**Comment by danielbayley** [comment]
> How about making them all align to the same baseline rather than just entering them vertically?

@jguddas @karsa-mistmere Fixed.

**Review by jguddas — APPROVED** [review]

**Review by karsa-mistmere — COMMENTED** [review]
`case-snake-upper` sticks out like a sore thumb because of the different letter `A` shape, but I'm not sure how it could be fixed. :/

**Comment by jguddas** [comment]
> `case-snake-upper` sticks out like a sore thumb because of the different letter `A` shape, but I'm not sure how it could be fixed. :/

We could change all the other A's to match :man_shrugging:

**Comment by danielbayley** [comment]
> `case-snake-upper` sticks out like a sore thumb because of the different letter `A` shape, but I'm not sure how it could be fixed. :/

@karsa-mistmere @jguddas What about something like these…?

preview

**Comment by jguddas** [comment]
> > `case-snake-upper` sticks out like a sore thumb because of the different letter `A` shape, but I'm not sure how it could be fixed. :/
> 
> Karsa Jakob Guddas What about something like these…?
> 
> preview

Looks terrible IMO.

**Comment by jguddas** [comment]
camel and lower case are also feel inconsistent.

**Pull request merged**